validator/project_validator.go

```diff
@@ -1951,6 +1952,40 @@ func validateGenerateTasks(p *model.Project) ValidationErrors {
 	return validateTimesCalledPerTask(p, ts, evergreen.GenerateTasksCommandName, 1, Error)
 }
 
+func validateModuleUsageInGitGetProject(p *model.Project) ValidationErrors {
+	var errs ValidationErrors
+
+	for _, bv := range p.BuildVariants {
```

Comment:
We have a function called `bvsWithTasksThatCallCommand`  that creates a mapping from build variants to tasks that run a given command. Can that (or that with some modification) be reused here?

Reply:
I just adapted it, it's a bit longer _but_ it now covers the edge case of pre & post functions which it did not before! 
I'm new to our project yaml and how users might use it, did not think of checking those! Thank you for the suggestion :)